lib/search.rb

```diff
@@ -280,6 +280,10 @@ def self.advanced_filters
     posts.where("topics.participant_count = 1")
   end
 
+  advanced_filter(/^status:unlisted$/) do |posts|
```

Comment:
I think for non staff "state:unlisted" should probably just return nothing? Mixed on this, technically an error is probably the best thing to do but that is too complex, I guess we can go with this behavior for now

Reply:
Yea I think we should just return and do nothing here. Less work to do on the DB side.

Reply:
In that case we should return `posts.where("1=0")` if user is not staff.